tst/hooks/sdk/useMediaStreamMetrics.test.tsx

```diff
@@ -0,0 +1,197 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { renderHook } from "@testing-library/react-hooks";
+import {
+  AudioVideoObserver,
+  Backoff,
+  ClientMetricReportDirection,
+  ClientMetricReportMediaType,
+  DefaultAudioVideoController,
+  DefaultAudioVideoFacade,
+  DefaultClientMetricReport,
+  DefaultReconnectController, DefaultVideoStreamIndex,
+  DefaultWebSocketAdapter,
+  Maybe,
+  MeetingSessionConfiguration, MeetingSessionCredentials, MeetingSessionURLs,
+  NoOpDebugLogger,
+  NoOpLogger,
+  NoOpMediaStreamBroker
+} from "amazon-chime-sdk-js";
+import React from "react";
+import useMediaStreamMetrics from "../../../src/hooks/sdk/useMediaStreamMetrics";
+import { AudioVideoContext } from "../../../src/providers/AudioVideoProvider";
+import NoOPAudioVideoFacade from "../../utils/NoOPAudioVideoFacade";
+import { act } from "@testing-library/react";
+import {
+  SdkIndexFrame,
+  SdkStreamDescriptor,
+  SdkStreamMediaType
+} from "amazon-chime-sdk-js/build/signalingprotocol/SignalingProtocol";
```

Comment:
I would try to avoid this kinda of import if possible :)

Reply:
`SdkIndexFrame,  SdkStreamDescriptor,  SdkStreamMediaType` are not exported from JS SDK so have to do it like this way.

Once we fix the error:
> Class constructor DefaultAudioVideoFacade cannot be invoked without 'new'

We could get rid of this.

Reply:
Removed as they are no longer needed.